### ISSUE TITLE
Redirects

### ISSUE THREAD

**Issue by lizzycederberg** [Threespot/aldf-wp#117]
@tedw David said there are about 600 existing redirects on aldf.org that will need to be setup on the new site. Is there a way to import those? Can he do it in the CMS?

(note: there were over 2,000 originally, but he doesn't want to keep the others)

**Comment by tedw** [comment]
Yes, I just installed the same redirect plugin they are using on the old site (which is a good one) so they can easily import the redirects.

https://live-aldf-wp.pantheonsite.io/wp/wp-admin/tools.php?page=redirection.php

**NOTE**: They were previously creating duplicate redirects for Salsa pages to add different tracking numbers. This can actually be done with a single redirect using the pattern below:

**Source**
`^/StopTheHunt\?id=(.*)`

**Target**
`https://org2.salsalabs.com/o/5154/p/salsa/donation/common/public/?donate_page_KEY=15859&track=$1`

Note: `$1` will be replaced with the id from the source URL

I would recommend this approach for any similar Engaging Networks links in the future.

**Comment by lizzycederberg** [comment]
David is taking care of all redirects. We will review them to make sure they look ok.
